fireapp/lib/presentation/unavailability_form/unavailability_form_widget.dart

```diff
@@ -0,0 +1,173 @@
+                        return ElevatedButton(
+                          style: ElevatedButton.styleFrom(
+                            minimumSize: const Size.fromHeight(40),
+                          ),
+                          onPressed: isFormValid ? viewModel.submitForm : null,
+                          child: Row(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: [
+                              Text(AppLocalizations.of(context)
+                                      ?.addUnavailabilityButton ??
+                                  "Add Schedule"),
+                              if (data is LoadingRequestState)
+                                SizedBox(
+                                  height: 8,
+                                  width: 8,
+                                  child: CircularProgressIndicator(
+                                      color: (Theme.of(context)
+                                              .primaryTextTheme
+                                              .titleLarge
+                                              ?.color ??
+                                          Colors.white)),
+                                )
+                            ],
+                          ),
+                        );
+                      },
+                    );
+                  }))
+        ],
+        children: [
+          Form(
+            key: viewModel.formKey,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                TextFormField(
+                  controller: viewModel.titleController,
+                  decoration: textFieldStylePositioned(
+                    context,
+                  ).copyWith(
+                      labelText: AppLocalizations.of(context)
+                              ?.enterUnavailabilityTitle ??
+                          "",
+                      prefixIcon: const Icon(Icons.event_note)),
+                  style: Theme.of(context).textTheme.labelLarge,
+                  validator: (v) => v!.isEmpty
+                      ? AppLocalizations.of(context)?.eventTitleEmptyError
+                      : null,
+                ),
+                StreamFormField<DateTime?>(
+                    stream: viewModel.selectedStartDate,
+                    builder: (context, value) {
+                      return DateFormField(
+                          currentValue: value,
+                          onValueChanged: viewModel.updateStartDate,
+                          decoration:
+                              textFieldStylePositioned(context).copyWith(
+                            prefixIcon: const Icon(Icons.calendar_today),
+                            labelText: AppLocalizations.of(context)
+                                ?.enterUnavailabilityStartDate,
+                          ));
```

Comment:
Ensure that closing parenthesis end on the same indentation as their opening counterpart (i.e put the second ")" on a new line). I know this feedback seems nitpicky, but declarative UI can quickly become difficult to read without consistent rules (though yours is very readable, I'm just trying to set up good habits)

Reply:
I agree. I've fixed it now, but I've found out that styling was coming from refactoring with "dart format". Do you know how to change that styling option?

Reply:
Dang, didn't realise that. There is literally no way to change dart's default styling settings